### ISSUE TITLE
FileLoad events are lost when the queue is paused due to an error on any file

### ISSUE THREAD

**Issue by chepemago** [CreateJS/PreloadJS#231]
Where there is an error and stopOnError is set to true, the queue is stopped but if some files are loaded at the same time, the event fileLoad is not dispatched.
If then, the queue is resumed, the files which were loaded but not "reported" are missed.

**Comment by lannymcnie** [comment]
What is the expectation here? If a file completes in a stopped queue, we currently ignore events. Maybe we should queue them up, and dispatch them ALL once the queue is restarted?

**Comment by chepemago** [comment]
Yes that can be one valid approach. The idea is that the user needs to
control which files have succeeded and which not.
